src/react/src/components/Contribute/ProductionLocationDialogCloseButton.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { bool, func } from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { makeProductionLocationCloseButtonStyles } from '../../util/styles';
+
+const ProductionLocationDialogCloseButton = withStyles(
+    makeProductionLocationCloseButtonStyles,
+)(props => {
+    const { isMobile, classes, handleShow } = props;
```

Comment:
Please wrap the `ProductionLocationDialogCloseButton` component with `withStyles` during the export and destructure props in the function parameters.